ReleaseTooling/Sources/FirebaseManifest/FirebaseManifest.swift

```diff
@@ -53,8 +53,8 @@ public let shared = Manifest(
     Pod("FirebasePerformance", platforms: ["ios", "tvos"], zip: true),
     Pod("FirebaseStorage", zip: true),
     Pod("FirebaseMLModelDownloader", isBeta: true, zip: true),
+    Pod("FirebaseVertexAI", zip: true),
```

Comment:
@andrewheard, what do you think? Should we re-order so the Firebase pod is pushed last or not?

Reply:
I don't feel too strongly either way. Moving it where you did will make it quicker to add Vertex AI to the `Firebase` pod if we ever find a compelling reason to. I'm fine with either and you've already moved it so 🚀 .